build_tools/provision_machine.py

```diff
@@ -200,7 +244,7 @@ def run(args):
     if args.run_id:
         retrieve_artifacts_by_run_id(args)
 
```

Comment:
Remove extra newline between `if` and `elif`?

```suggestion
    if args.run_id:
        retrieve_artifacts_by_run_id(args)
```